test/modules/packages/collections/CollectionNQueens.chpl

```diff
@@ -156,9 +156,10 @@ coforall loc in Locales do on loc {
   coforall tid in 0 .. #here.maxTaskPar {
     barrier.barrier();
     var nSpins : int;
+    // BUG: If we do not specify the type here, we get a compiler internal error...

```

Comment:
Is there already an issue for this bug? If not, I'll add one

Also.. sorry that you've hit so many bugs :(

Reply:
It is an issue, but I didn't have time to figure out what the cause was. It had something to do with 'Visibility Blocks', and I couldn't figure out why, all I know is that it cropped up around the time I converted `DistributedDeque` to record wrapping. 

Also no problem, occupational hazards and all that.